datatables.go

```diff
@@ -466,3 +466,52 @@ func (p *Parser) infernoExpired(inf *common.Inferno) {
 
 	delete(p.gameState.infernos, inf.EntityID)
 }
+
+func (p *Parser) bindGameRules() {
+	grPrefix := func(s string) string {
```

Comment:
What's your opinion on replacing this with `gameRulesPrefix + "m_gamePhase"` (or `m_totalRoundsPlayed`, respectively)?

Might be a bit clearer to see what's going on at a glance. If you agree I'd move the dot `.` into `gameRulesPrefix` too.

Reply:
My personal preference is using a function because I feel like it makes it harder to mess up the formatting, as well as being more readable to me. But that's entirely subjective and probably just because it's the way that I'm used to seeing it. Either way works for me :slightly_smiling_face: